src/lib/utils/meta_table_manager.cpp

```diff
@@ -229,6 +271,26 @@ std::shared_ptr<Table> MetaTableManager::generate_accurate_segments_table() {
   return output_table;
 }
 
+std::shared_ptr<Table> MetaTableManager::generate_plugins_table() {
+  const auto columns = TableColumnDefinitions{{"plugin_name", DataType::String, false}};
+
+  auto output_table = std::make_shared<Table>(columns, TableType::Data, std::nullopt, UseMvcc::Yes);
+
+  for (const auto& plugin : Hyrise::get().plugin_manager.plugin_names()) {
+    output_table->append({pmr_string{plugin}});
+  }
+
+  return output_table;
+}
+
+void MetaTableManager::_insert_into_plugins(const std::string& value) {
+  Hyrise::get().plugin_manager.load_plugin(value + DYNAMIC_LIBRARY_SUFFIX);
+}
+
+void MetaTableManager::_delete_from_plugins(const std::string& value) {
+  Hyrise::get().plugin_manager.unload_plugin(value);
+}
```

Comment:
Not sure how we end up here. Insert and delete operate on table. The mechanism for converting a table modification to a method like this should be generic. Things that come to mind:

* Tables have multiple columns. While we only use a single field here, we should plan for that.
* Tables have multiple rows. We should be able to do `UPDATE meta_memory_budgets SET budget = budget * 2 WHERE ...`.

Maybe we should do some brainstorming next Wednesday on what settings we want to set this way. Not necessarily because we need to implement them right now, but because we should make sure that whatever we come up with here doesn't block us in the future.

Reply:
Since next week is the TaC II block week, do you also have some time later on today?

Reply:
Sure. Maybe 13:00, also pinging @Bouncner and @Bensk1 (and anyone else who might care)

Reply:
Alright!